src/rhsmlib/services/register.py

```diff
@@ -44,6 +45,16 @@ def register(self, org, activation_keys=None, environment=None, force=None, name
         if kwargs:
             raise exceptions.ValidationError(_("Unknown arguments: %s") % kwargs.keys())
 
+        if no_insights is True:
+            try:
+                with open('/dev/null', 'w') as devnull:
+                    subprocess.call(['/usr/bin/systemctl', 'mask', '--now', 'insights-register.path'], stdout=devnull,
+                                    stderr=devnull)
+            except:
```

Comment:
I would change this generic exception handling to this:
```python
exception Exception as err:
```
Then I would display err in log.warning message.

Reply:
Done